deployment/helm/synopsys-alert/values.yaml

```diff
@@ -41,6 +41,29 @@ cfssl:
   securityContext: {}
   podSecurityContext: {}
 
+# alertdb - configurations for the alertdb Pod
+alertdb:
+  registry: "docker.io/centos" # override the docker registry at container level
+  isExternal: true # false for running Postgres as a container and true for using External Postgres database
+  host: "localhost" # required only for external postgres, for postgres as a container, it will point to <name>-alert-postgres
+  port: 5432
+  # NOTE: do not change usernames when using postgres as a container; only configure if using external database
+  userUserName: sa
```

Comment:
you won't be having 2 users in an app? one is an admin who can have access at DDL level and another user is having DML level access. The code need to use the DML level access. This is to avoid any SQL injection and also the permissions restricting to the user.

Reply:
At the moment no.  We would need to test using the liquibase user and password properties for DDL access and the spring datasource properties for the DML access.  I will file a ticket for this.